ember-phone-input/src/components/phone-input.ts

```diff
@@ -100,10 +174,10 @@ export default class PhoneInputComponent extends Component {
    * country. Possible values are 'polite', 'aggressive' and 'off'. Defaults to
    * 'polite'.
    * @argument autoPlaceholder
-   * @type {string}
+   * @type {'aggressive' | 'off' | 'polite' | undefined}
    */
 
-  get autoPlaceholder() {
+  get autoPlaceholder(): 'aggressive' | 'off' | 'polite' | undefined {
```

Comment:
It's passed down to intl-tel-input setup as options so the idea was to just have the same type as they do to avoid discrepancy

Reply:
I'll resolve this since I strongly think we should follow the `intl-tel-input` options types since our component's arguments are only used to pass down to the `intl-tel-input` component so using its types makes more sense. Let me know if you think this requires further discussion